tools/wpt/virtualenv.py

```diff
@@ -9,11 +9,13 @@
 logger = logging.getLogger(__name__)
 
 class Virtualenv(object):
-    def __init__(self, path):
+    def __init__(self, path, skip_virtualenv_setup):
         self.path = path
-        self.virtualenv = find_executable("virtualenv")
-        if not self.virtualenv:
-            raise ValueError("virtualenv must be installed and on the PATH")
+        self.skip_virtualenv_setup = skip_virtualenv_setup
+        if not skip_virtualenv_setup:
+            self.virtualenv = find_executable("virtualenv")
```

Comment:
So this sort of leaves the class in a semi-initalised state. I guess it's OK for now, but it doesn't seem perfect.